localstack/services/cloudformation/engine/template_deployer.py

```diff
@@ -647,6 +650,7 @@ def get_resource_model_instance(resource_id: str, stack) -> Optional[GenericBase
     canonical_type = canonical_resource_type(resource_type)
     resource_class = RESOURCE_MODELS.get(canonical_type)
     if not resource_class:
+        # usage.missing_resource_types.record(canonical_type)
```

Comment:
Kinda. It's a bit unclear actually in CFn right now where it's best to put the logging, just because the template deployer is still so convoluted. I'll remove it for now though. We'll need to re-evaluate the placement after an initial rework of the deployer